electron/main.ts

```diff
@@ -141,3 +143,25 @@ app.whenReady().then(() => {
     }
   });
 });
+
+const args = parseArgs(process.argv.slice(1));
```

Comment:
we should support the passing of a folder when launching, which would automatically  open that folder (i.e. setRootFolder..) in the UI.. i.e. as a user I should be able to type

`monokle . `

or 

`monokle manifests`

Reply:
yeah, I can get it right now but I didn't send it yet. I am reading the code first. I need to figure out how we load a directory. As you see I get the parameter of `executed-from`. I am planning to send it inside react and handle it there.

![Screen Shot 2021-08-19 at 10 54 51](https://user-images.githubusercontent.com/2957624/130030136-df04da8c-2090-4238-aedf-ba9676b1548e.png)

Reply:
ok great !